gitlab_total_repo_count

```diff
@@ -30,7 +33,13 @@ try:
         repos = os.listdir(subdir)
         for repo in repos:
             if repo.endswith('.wiki.git'):
-                wiki_count += 1
+                try:
+                    if os.listdir(subdir + os.sep + repo + '/objects/pack'):
+                        wiki_count += 1
+                    else:
+                        wiki_unused_count += 1
+                except OSError as e:
+                    pass
```

Comment:
In which case does this error occur? Is this also a case for "unused"?

Reply:
In the OSError case that directory will be broken, so we should not count it, I think. What do you think?

Reply:
I'm not sure about it.

Another possible action: Raise the error to inform the sysadmin about the broken directory. Alternative: A new counter with "faulty repositories"? What's your opinion about that?

Reply:
While my original opinion was `Do nothing (neither count nor inform)`, now `Count as empty/unused (because it can't be used if it's faulty)` looks good to me.